mizani/breaks.py

```diff
@@ -118,7 +118,7 @@ def _log_breaks(limits):
             return base ** _min
 
         step = (_max-_min)//n + 1
-        return base ** np.arange(_min, _max+1, step)
+        return base ** np.arange(_min, _max+1, step, dtype=float)
```

Comment:
Yes you are right, and I do not think the tests would have caught that. How about this

```python
dtype = float if (_min < 0) else int
base ** np.arange(_min, _max+1, step, dtype=dtype)
```